go/kbfs/kbfsedits/user_history.go

```diff
@@ -161,10 +185,14 @@ func (uh *UserHistory) Get(loggedInUser string) (
 
 		// Break it up into individual clusters
 		for _, wh := range history.History {
-			if len(wh.Edits) > 0 {
+			if len(wh.Edits) > 0 || len(wh.Deletes) > 0 {
+				var serverTime keybase1.Time
+				if len(wh.Edits) > 0 {
+					serverTime = wh.Edits[0].ServerTime
+				}
 				clusters = append(clusters, keybase1.FSFolderEditHistory{
 					Folder:     history.Folder,
-					ServerTime: wh.Edits[0].ServerTime,
+					ServerTime: serverTime,
```

Comment:
displaying time 0 if there are only deletes seems a little icky... not that I have a better suggestion of what time to display, but it. might be confusing.

Reply:
Do we show it in the UI as something other than Jan 1970 or Jan 1 year 0000?

Reply:
Hmm, I don't know if this time is shown anywhere in the GUI, if there are no edits in the folder.  If you only have deletes, I don't think the GUI will currently show anything.

Note that this is just the overall time of this cluster of edits -- it's mainly used for sorting.  Each edit/delete still has its own timestamp that can be displayed by the UI.